### ISSUE TITLE
packages.models.MultipleObjectsReturned: get() returned more than one Erratum -- it returned 2!

### ISSUE THREAD

**Issue by IPv777** [furlongm/patchman#270]
Hello :)
I get this when using `patchman -a`, an idea ?

Thanks ! :)

```
Downloading Errata Checksum:       100%|##############################################|Time: 0:00:00
Downloading CentOS Errata:         100%|##############################################|Time: 0:00:00
Processing 9792 Errata:              35%|################                              |ETA:  0:00:49
Traceback (most recent call last):  35%|################                              |ETA:  0:00:49
  File "/usr/bin/patchman", line 674, in <module>
    main()
  File "/usr/bin/patchman", line 668, in main
    showhelp = process_args(args)
  File "/usr/bin/patchman", line 658, in process_args
    update_errata(args.force)
  File "/usr/lib/python2.7/dist-packages/packages/utils.py", line 91, in update_errata
    parse_errata(bunzip2(data), force)
  File "/usr/lib/python2.7/dist-packages/packages/utils.py", line 129, in parse_errata
    e = parse_errata_tag(child.tag, child.attrib, force)
  File "/usr/lib/python2.7/dist-packages/packages/utils.py", line 153, in parse_errata_tag
    force=force)
  File "/usr/lib/python2.7/dist-packages/packages/utils.py", line 214, in create_erratum
    synopsis=synopsis)
  File "/usr/lib/python2.7/dist-packages/django/db/models/query.py", line 464, in get_or_create
    return self.get(**lookup), False
  File "/usr/lib/python2.7/dist-packages/django/db/models/query.py", line 384, in get
    (self.model._meta.object_name, num)
packages.models.MultipleObjectsReturned: get() returned more than one Erratum -- it returned 2!

```

**Comment by furlongm** [comment]
This is fixed on the master branch by e235437ae7cab1a2db4a3eb0df979da661a3d6af

I will try to get a release out before the end of the year.

**Comment by IPv777** [comment]
thanks <3
